cfgov/v1/tests/management/commands/test_update_data_snapshot_values.py

```diff
@@ -0,0 +1,37 @@
+from django.core.management import call_command
+from django.test import Client, TestCase
+
+from wagtail.wagtailcore.blocks import StreamValue
+
+from scripts import _atomic_helpers as atomic
+from v1.models.browse_page import BrowsePage
+from v1.tests.wagtail_pages.helpers import publish_page
+
+django_client = Client()
```

Comment:
This is unnecessary since your test case inherits from `django.test.TestCase`, as it automatically [gets one as `self.client`](https://docs.djangoproject.com/en/1.8/topics/testing/tools/#django.test.SimpleTestCase.client). The `Client` class also maintains some state between requests (like cookies and such) so defining it outside of the tests like this could potentially allow tests to affect each other in some subtle way.